src/corpus/ui/PopoverAlignment.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import TextService from 'corpus/application/TextService'
+import { ChapterId } from 'transliteration/domain/chapter-id'
+import withData from 'http/withData'
+import { LineDetails } from 'corpus/domain/line-details'
+
+export const LineInfoContext = React.createContext<
+  | {
+      chapterId: ChapterId
+      lineNumber: number
+      variantNumber: number
+      textService: TextService
+    }
+  | Record<string, never>
+>({})
+
+const AlignedManuscriptTokens = withData<
```

Comment:
This is almost identical to `Score`.

Reply:
This is not the place to [useContext](https://reactjs.org/docs/context.html).